packages/newsroom-signup/src/DataWrapper.tsx

```diff
@@ -81,16 +71,25 @@ export class DataWrapper extends React.Component<DataWrapperProps> {
                       return (
                         <Mutation mutation={saveCharterMutation}>
                           {(saveCharter: MutationFunc) => {
-                            return this.props.children({
-                              grantRequested,
-                              grantApproved,
-                              newsroomDeployTx,
-                              newsroomAddress,
-                              profileWalletAddress: data.currentUser.ethAddress,
-                              persistedCharter,
-                              saveAddress,
-                              persistCharter: this.saveCharterFuncFromMutation(saveCharter),
-                            });
+                            return (
+                              <Mutation mutation={saveStepsMutation}>
```

Comment:
This now has 2 nested <Query>'s and 3 nested <Mutations>'s, needs refactoring with apollo's compose and HOCs, will do this after launch.

Reply:
👍